gamemode/modules/chat/sv_chat.lua

```diff
@@ -0,0 +1,27 @@
+function RVR.canHear( listener, speaker )
+    if not listener:Alive() or not speaker:Alive() then
+        return false
+    end
+    
+    local speakerPos = speaker:GetPos()
+    local listenerPos = listener:GetPos()
+
+    if listenerPos:Distance( speakerPos ) > GAMEMODE.Config.Chat.CHAT_DISTANCE then
```

Comment:
Idk how much this matters, but I often see people using `DistToSqr` over `Distance` and squaring the config value, as apparently that's faster. Up to you as to if it's worth the slightly less readable code.

Reply:
i dont think efficiency will be an issue here as this is only ran once on chat